src/locales/en/translation.json

```diff
@@ -1649,7 +1649,7 @@
       },
       "section2": {
         "subHeading" : "What you can do",
-        "paragraph1" : "If your passport has expired, you can still use it to prove your identity up until 18 months after the expiry date."
+        "paragraph1" : "Continue to the service you were trying to use and look for other ways to prove your identity."
```

Comment:
Having just searched my development machine I can see the new content is already present in the Welsh translation file